multires-lyrics-search/app.py

```diff
@@ -42,7 +43,8 @@ def input_fn():
 def index():
     f = Flow.load_config('flows/index.yml')
     with f:
-        f.index(input_fn, request_size=8)
+        with TimeContext(f'QPS: indexing', logger=f.logger):
```

Comment:
same, specify the nr of docs

Reply:
hmm but I dont know it upfront in this case do I? We use the generator `input_fn()  ` based on some csv file.

Reply:
You should look at that file and count the lines maybe. Or make the generator into a list and get its `len()`. (yes, not efficient, but needed in this case)